### DISCUSSION TITLE
Bundled OpenSSL versions?

### DISCUSSION THREAD

**Discussion by apereira-dcca** [git-for-windows/git#4297] [General]
I'm curious how quickly new versions of OpenSSL will get incorporated in the distribution of git-for-windows? Currently, the 2.39.2 installer is pushing 1.1.1s, but 1.1.1t was released on 7 February 2023 to address security vulnerabilities.

**Comment by rimrul** [comment]
OpenSSL 1.1.1t was included into the SDK on February 8th with #4277. It will be included with Git for Windows 2.40.0, which is expected for March 13th or a few days after that.
